pipeline/config-prod.groovy

```diff
@@ -273,7 +273,7 @@ environments {
         vars {
             DB_PVC_SIZE = '50Gi'
             DOCUMENT_PVC_SIZE = '50Gi'
```

Comment:
should we take this change to increase document storage pvc? how much room do we have left right now.

Reply:
We have used 5.5GB of the 50GB. Not a pressing concern but more of a mid-term issue.